lib/openh264.py

```diff
@@ -41,9 +41,10 @@ def EncodeCommandLine(self, parameters, bitrate, videofile, encodedfile):
         '-bf %s '
         '-sw %d -sh %d '
         '-dw 0 %d -dh 0 %d '
-        '-rc 0 -maxbrTotal %d -tarb %d '
+        '-rc 0 -maxbrTotal 0 -tarb %d '
```

Comment:
Can we remove the command line parameter totally, or will removing it cause an inappropriate value from the config file to be used?